README.md

```diff
@@ -1231,7 +1231,7 @@ Expects the following parameters:
 
 - conn ([`Connection`](#connection))
 
-- storedQuery ([`StoredQueryOptions`](#storedqueryoptions))
+- storedQuery ([`StoredQueryOptions | object`](#storedqueryoptions | object))
```

Comment:
The hash is a link within the document which is broken now (other location is as well). It should be:
```
([`StoredQueryOptions`](#storedqueryoptions) | object)
```

Reply:
oof this is probably a bug elsewhere too then... let's file a separate ticket for fixing the docs generation